pkg/strategy/supertrend/draw.go

```diff
@@ -0,0 +1,91 @@
+package supertrend
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/c9s/bbgo/pkg/bbgo"
+	"github.com/c9s/bbgo/pkg/interact"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/wcharczuk/go-chart/v2"
+)
+
+func (s *Strategy) InitDrawCommands(store *bbgo.SerialMarketDataStore, profit, cumProfit types.Series) {
+	bbgo.RegisterCommand("/pnl", "Draw PNL(%) per trade", func(reply interact.Reply) {
+		canvas := s.DrawPNL(profit)
+		var buffer bytes.Buffer
+		if err := canvas.Render(chart.PNG, &buffer); err != nil {
+			log.WithError(err).Errorf("cannot render pnl in drift")
+			reply.Message(fmt.Sprintf("[error] cannot render pnl in ewo: %v", err))
+			return
+		}
+		bbgo.SendPhoto(&buffer)
+	})
+	bbgo.RegisterCommand("/cumpnl", "Draw Cummulative PNL(Quote)", func(reply interact.Reply) {
+		canvas := s.DrawCumPNL(cumProfit)
+		var buffer bytes.Buffer
+		if err := canvas.Render(chart.PNG, &buffer); err != nil {
+			log.WithError(err).Errorf("cannot render cumpnl in drift")
+			reply.Message(fmt.Sprintf("[error] canot render cumpnl in drift: %v", err))
+			return
+		}
+		bbgo.SendPhoto(&buffer)
+	})
+}
+
+func (s *Strategy) DrawPNL(profit types.Series) *types.Canvas {
```

Comment:
pass instanceID from the func argument so this can be a simple function?

Reply:
Done